### PR TITLE
Get ETag from response header instead of data

### PR DESCRIPTION
**Pull request by oBlissing**
Needed for [LXL-2178](https://jira.kb.se/browse/LXL-2178) to work.

### CODE REVIEW

File: viewer/v2client/src/components/Inspector.vue [hunk 1]
```diff
@@ -151,6 +152,7 @@ export default {
       const fetchUrl = `${this.settings.apiPath}/${this.documentId}/data.jsonld?${randomHash}`;
       fetch(fetchUrl).then((response) => {
         if (response.status === 200) {
+          this.documentETag = response.headers.get('ETag').replace(/"/g, ''); // Fix double quoted string
```
Comment (mxtthias) [inline viewer/v2client/src/components/Inspector.vue]: Why strip quoting?

Comment (mxtthias) [reply viewer/v2client/src/components/Inspector.vue]: Backend sends a quoted string in the ETag header. I would have assumed that frontend should treat it as an opaque value and not manupulate it in any way?

Comment (oBlissing) [reply viewer/v2client/src/components/Inspector.vue]: Got it, will fix and update the PR.

Comment (niklasl) [reply viewer/v2client/src/components/Inspector.vue]: The keys is to make the subsequent `If-None-Match` identical. So if we get:
```
    ETag: "abc"
```
Do we send:
```
   If-None-Match: "abc"
```
?

**Pull request merged**